specification/netapp/resource-manager/Microsoft.NetApp/stable/2019-05-01/netapp.json

```diff
@@ -1372,6 +1372,10 @@
     "poolProperties": {
       "description": "Pool properties",
       "type": "object",
+      "required": [
```

Comment:
@leonardbf  This seems to be a breaking change... Is this model used for a request?

Reply:
> @leonardbf This seems to be a breaking change... Is this model used for a request?

@praries880 yes this is used for requests, pool creation for instance. The fact is that the required field here was missing, i.e. the swagger did not correctly represent the resource provider. This was therefore a correction.